src/api/apps/articles/model/sanitize.js

```diff
@@ -19,6 +20,12 @@ export const sanitizeLink = urlString => {
   if (url.hostname.includes(".artsy.net")) {
     url.protocol = "https:"
   }
+  if (url.href.includes("artsy.net/post/")) {
+    url.href = url.href.replace("/post/", "/article/")
+  }
+  if (url.href.includes("/posts")) {
```

Comment:
This looks like an unsafe replace-- we should use `"artsy.net/posts/"` like above to ensure we don't change non-artsy URLS

Reply:
```suggestion
  if (url.href.includes("artsy.net/posts")) {
```